features/growable-sharedarraybuffer.yml

```
@@ -0,0 +1,8 @@
+name: Growable SharedArrayBuffer
+description: ShareArrayBuffer objects can take a maximum length allowing them to grow in place.
```

The description here is a bit circular ("What's a growable buffer?" "A buffer that grows."). What do you think of something like this?

```suggestion
description: The `grow()` method of a `SharedArrayBuffer`, constructed with the `maxByteLength` option, increases the size of the buffer in place.
```